build-package.sh

```diff
@@ -583,7 +583,7 @@ while (($# >= 1)); do
 			;;
 		-L) export TERMUX_GLOBAL_LIBRARY=true;;
 		-q) export TERMUX_QUIET_BUILD=true;;
-		-Q) set -x;;
+		-Q) export PS4='+$0 \[\e[32m\]${FUNCNAME[0]:-<global scope>}${FUNCNAME[*]:+()}:$LINENO\[\e[0m\] '; set -x;;
```

Comment:
The tracing output for `build-packages.sh -Q` looks like this.
<img width="1918" height="860" alt="Image" src="https://github.com/user-attachments/assets/d9a02a67-a091-46ae-867f-a127a630d3cd" />